controllers/dscinitialization/suite_test.go

```diff
@@ -48,12 +73,17 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
+	ctx, cancel = context.WithCancel(context.TODO())
 	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
-
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
-		CRDDirectoryPaths:     []string{filepath.Join("..", "..", "config", "crd", "bases")},
+		CRDDirectoryPaths: []string{
+			filepath.Join("..", "..", "config", "crd", "bases"),
+			filepath.Join("..", "..", "config", "crd", "external"),
+		},
 		ErrorIfCRDPathMissing: true,
+		// add this config for arm machines
+		//BinaryAssetsDirectory: "~/Library/Application Support/io.kubebuilder.envtest/k8s/{$VERSION}-darwin-arm64",
```

Comment:
If it's important to support arm64 as test env, we can conditionally add it to `envtest.Environment` based on the golang built-in platform env var `runtime.GOOS` instead of having it as a comment.

Reply:
As for this part, I think the users can follow the doc here: https://book.kubebuilder.io/reference/envtest.html#configuring-envtest-for-integration-tests

Reply:
Fair enough, but then I would move this hint to a doc where testing is described.